Failing test: module 'pydp' has no attribute 'status_code_to_string'

When I run `make test-all` I see 10 passing tests and 1 fail:
```
base/test_logging.py .....                                                                                                                                                                               [ 45%]
base/test_status.py ..F...                                                                                                                                                                               [100%]

=================================================================================================== FAILURES ===================================================================================================
________________________________________________________________________________________ TestStatus.test_code_to_string ________________________________________________________________________________________

self = <tests.base.test_status.TestStatus object at 0x7f438b720588>

>   ???
E   AttributeError: module 'pydp' has no attribute 'status_code_to_string'

/home/lukasz/src/OpenMind/PyDP/tests/base/test_status.py:39: AttributeError
```

### Steps to Reproduce Behavior

Run `make test-all`.

### System information

* PyDP version: commit b6abd752058062c34a84c1e2bce3dd61b64460de
* Python version: 3.6.9
* Bazel version: 2.1.0
* Operating System: Ubuntu 18.04

--- a/tests/base/test_status.py
+++ b/tests/base/test_status.py
@@ -5,28 +5,28 @@ import pydp as pd
 
 class TestStatus():
     def test_hello(self):
-        i =1 
+        i =1
         assert 1 == i
 
     def test_status_codes(self):
         status_code_available = [
-            "StatusCode.kOk", 
-            "StatusCode.kCancelled", 
-            "StatusCode.kUnknown", 
+            "StatusCode.kOk",
+            "StatusCode.kCancelled",
+            "StatusCode.kUnknown",
             "StatusCode.kInvalidArgument",
-            "StatusCode.kDeadlineExceeded", 
-            "StatusCode.kNotFound", 
-            "StatusCode.kAlreadyExists", 
-            "StatusCode.kPermissionDenied", 
-            "StatusCode.kResourceExhausted", 
+            "StatusCode.kDeadlineExceeded",
+            "StatusCode.kNotFound",
+            "StatusCode.kAlreadyExists",
+            "StatusCode.kPermissionDenied",
+            "StatusCode.kResourceExhausted",
             "StatusCode.kFailedPrecondition",
-            "StatusCode.kAborted", 
+            "StatusCode.kAborted",
             "StatusCode.kOutOfRange",
             "StatusCode.kUnimplemented",
-            "StatusCode.kInternal", 
-            "StatusCode.kUnavailable", 
-            "StatusCode.kDataLoss", 
-            "StatusCode.kUnauthenticated", 
+            "StatusCode.kInternal",
+            "StatusCode.kUnavailable",
+            "StatusCode.kDataLoss",
+            "StatusCode.kUnauthenticated",
         ]
         actual_codes = []
         x = range(17)
@@ -34,9 +34,9 @@ class TestStatus():
             actual_codes.append(str(pd.Status.StatusCode(n)))
         print(actual_codes)
         assert status_code_available == actual_codes
-    
+
     def test_code_to_string(self):
-        s = pd.status_code_to_string(pd.Status.StatusCode(3))
+        s = pd.Status.status_code_to_string(pd.Status.StatusCode(3))
         assert s == "kInvalidArgument"
 
 class TestSampleLoad():
@@ -47,7 +47,7 @@ class TestSampleLoad():
         payload_content = "example payload content"
         s.set_payload(url, payload_content)
         assert s.get_payload(url) == None
-    
+
     def test_payload_test_1(self):
         # in all Status code except 0, payload is added
         s = pd.Status(pd.Status.StatusCode(1), "New status object")

